pkgs/development/libraries/x265/default.nix

```diff
@@ -66,7 +67,9 @@ stdenv.mkDerivation rec {
     })
     # Fix detection of NEON (and armv6 build) :
     ./fix-neon-detection.patch
-  ];
+  ]
```

Comment:
alternatively just remove `.endfunc` and `.func`. they have no effect on the assembled code anyway as they are pseudo opts related to symbol generation. one can verify this by assembling with gas adding and removing the ops. 

```diff
diff --git a/pkgs/development/libraries/x265/default.nix b/pkgs/development/libraries/x265/default.nix
index 7f44834d4867..4285aac6e3c3 100644
--- a/pkgs/development/libraries/x265/default.nix
+++ b/pkgs/development/libraries/x265/default.nix
@@ -72,6 +72,10 @@ stdenv.mkDerivation rec {
     substituteInPlace cmake/Version.cmake \
       --replace "unknown" "${version}" \
       --replace "0.0" "${version}"
+
+    for file in common/{aarch64,arm}/asm.S; do
+      sed -i '/\.\(end\)\?func/d' $file
+    done
   ''
   # There is broken and complicated logic when setting X265_LATEST_TAG for
   # mingwW64 builds. This bypasses the logic by setting it at the end of the
```

Reply:
see also: https://android.googlesource.com/platform/ndk/+/master/docs/ClangMigration.md#no